Azure/sonic-utilities
# Add command line support for thermal control

**- What I did**
1. Add a "show platform fanstatus" command to allow user fetch FAN status data.
2. Add a "show platform temperature" command to allow user fetch thermal status data.

**- How I did it**
1. Add a fanshow script to get FAN status from database and display it in command line
2. Add a tempershow script to get thermal status from database and display it in command line
3. Add two sub-commands under "platform" group

**- How to verify it**
1. Issue "show platform fanstatus" in command line and see the output.
2. Issue "show platform temperature" in command line and see the output.

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**
